.github/workflows/run_tests.yml

```diff
@@ -21,7 +18,7 @@ jobs:
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
         with:
-          python-version: ${{ matrix.python-version }}
+          python-version: 3.9
```

Comment:
This version isn't massively important, as the tests run inside a Docker container.